fuzz/fuzz_targets/parse-conditions.rs

```diff
@@ -5,12 +5,12 @@ use clvmr::allocator::Allocator;
 use chia::fuzzing_utils::{BitCursor, make_tree};
 use chia::gen::conditions::parse_spends;
 
-use chia::gen::flags::{COND_ARGS_NIL, STRICT_ARGS_COUNT, NO_UNKNOWN_CONDS};
+use chia::gen::flags::{COND_ARGS_NIL, STRICT_ARGS_COUNT, ENABLE_ASSERT_BEFORE};
```

Comment:
Why remove `NO_UNKNOWN_CONDS`?

Reply:
good question. I believe I was thinking that it would increase the test surface of the fuzzer. Thinking about it again though, it probably just shifts it a little bit from the failure case to the ignore-case. I realize I ended up removing a fuzzing case. I'll restore this.

Reply:
I believe this was a copy-paste error actually. fixed now